grafana/dashboards/cmode/volume.json

```diff
@@ -6457,18 +6457,417 @@
           "type": "timeseries"
         }
       ],
-      "title": "Volume Capacity %",
+      "title": "Capacity %",
       "type": "row"
     },
     {
       "collapsed": true,
-      "datasource": "${DS_PROMETHEUS}",
+      "datasource": null,
       "gridPos": {
         "h": 1,
         "w": 24,
         "x": 0,
         "y": 20
       },
+      "id": 153,
+      "panels": [
+        {
+          "datasource": "${DS_PROMETHEUS}",
+          "gridPos": {
+            "h": 2,
+            "w": 24,
+            "x": 0,
+            "y": 21
+          },
+          "id": 155,
+          "options": {
+            "content": "<br>\n\nTop clients panels are disabled by default and are supported by the Volume Template in RestPerf Collector. For more details on how to enable them, please refer to the discussion at https://github.com/NetApp/harvest/discussions/3130.",
+            "mode": "markdown"
```

Comment:
We can add warning something like: Enabling this may slow down data collection due to a high number of rest calls and post processing for top clients.